### ISSUE TITLE
OnAwsCloudEnvironmentCondition slows down application startup

### ISSUE THREAD

**Issue by sbuettner** [spring-cloud/spring-cloud-aws#181]
The `OnAwsCloudEnvironmentCondition` leads to calls to ` AwsCloudEnvironmentCheckUtils.isRunningOnCloudEnvironment()` which itself leads to calls to `EC2MetadataUtils.getData()` even though the application is launched locally. The internal retry mechanism lead to multiple ec2 meta info calls during the launch of the application which increases the startup time heavily:
```
2016-10-31 15:30:22.353 DEBUG 54185 --- [  restartedMain] c.a.internal.EC2CredentialsUtils         : An IOException occured when connecting to service endpoint: http://169.254.169.254/latest/meta-data/instance-id
 Retrying to connect again.
2016-10-31 15:30:24.359 DEBUG 54185 --- [  restartedMain] c.a.internal.EC2CredentialsUtils         : An IOException occured when connecting to service endpoint: http://169.254.169.254/latest/meta-data/instance-id
 Retrying to connect again.
2016-10-31 15:30:26.363 DEBUG 54185 --- [  restartedMain] c.a.internal.EC2CredentialsUtils         : An IOException occured when connecting to service endpoint: http://169.254.169.254/latest/meta-data/instance-id
 Retrying to connect again.
2016-10-31 15:30:28.367 DEBUG 54185 --- [  restartedMain] c.a.internal.EC2CredentialsUtils         : An IOException occured when connecting to service endpoint: http://169.254.169.254/latest/meta-data/instance-id
 Retrying to connect again.
2016-10-31 15:30:30.373 DEBUG 54185 --- [  restartedMain] c.a.internal.EC2CredentialsUtils         : An IOException occured when connecting to service endpoint: http://169.254.169.254/latest/meta-data/instance-id
 Retrying to connect again.
```

It would be great to disable/override this process using a property.

**Comment by magdenkov** [comment]
Having same issue...

**Comment by mrtduzgun** [comment]
This fixed on aws-sdk v1.11.25. [https://github.com/aws/aws-sdk-java/issues/799](url)
